test/dispatcher.js

```diff
@@ -62,7 +75,7 @@ suite('Event Generation and Dispatching', function() {
 
   test('PointerEvents from mouse fire anywhere by default', function() {
     // move always fires
-    var cb = chai.spy();
+    var cb = chai.spy;
```

Comment:
I don't see where you're loading the spy functionality. The existing tests have to load that separately: https://github.com/jquery/PEP/blob/67d1dcad49774522ea8bd852b03d6d535883567c/test/runner.html#L10.